pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/MainDesignerController.java

```diff
@@ -354,7 +356,7 @@ private void onOpenFileClicked() {
     private void loadSourceFromFile(File file) {
         if (file != null) {
             try {
-                String source = IOUtils.toString(new FileInputStream(file));
+                String source = IOUtils.toString(new FileInputStream(file), StandardCharsets.UTF_8);
```

Comment:
That's the other change, that is outside of test code.
Before we were using the current platform's default encoding, now we use UTF-8. Both behaviors might be wrong. But I think, using UTF-8 is safe (I would consider this the default nowadays).
We might need to add a configuration option in the file-open-dialog at some time, but that's definitely out of scope for this PR.

Reply:
Agreed, this is likely outside the scope of the PR.